aws-dynamodb-point-in-time-recovery-disabled.yaml

```
@@ -0,0 +1,37 @@
+rules:
+- id: aws-dynamodb-point-in-time-recovery-disabled
+  patterns:
+  - pattern: |
+      resource "aws_dynamodb_table" $ANYTHING {
+        ...
+      }
+  - pattern-not-inside: |
+      resource "aws_dynamodb_table" $ANYTHING {
+        ...
+        point_in_time_recovery {
+          enabled = true
+        }
+        ...
+      }
+  message: Point-in-time recovery is not enabled for the DynamoDB table. DynamoDB
+    tables should be protected against accidental or malicious write/delete actions.
+    By enabling point-in-time-recovery you can restore to a known point in the event
+    of loss of data.
+  languages:
+  - hcl
+  severity: WARNING
+  metadata:
+    category: security
```

```suggestion
  metadata:
    category: security
    references: 
```
This rule is missing references. You could link to the relevant section in the docs:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/dynamodb_table#point_in_time_recovery